docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+# Contributing to Threadbare
+
+We welcome contributions to Threadbare!
+
+## Language
+
+In-game text and dialogue for the main storyline should be written in English,
+for translation into other languages. Learner-contributed text and dialogue may
+be written in another language if preferred.
+
+Prefer US English spellings for in-game text and dialogue, as well as source
+code, comments, and commit messages:
+
+- “Color”, not “colour”
+- “Traveler”, not “traveller”
+
+Use “dialogue” when referring to in-game speech (following the `DialogueManager`
+API), but “dialog” if referring to a dialog box.
+
+## Coding style
+
+GDScript source code should follow the [GDScript style guide][]. This is
+enforced using `gdlint` and `gdformat` from [godot-gdscript-toolkit][] when
+changes are submitted to the project.
+
+You can use [pre-commit][] to run the same checks locally. Install `pre-commit`
+then run the following command in your Threadbare checkout:
+
+```
+pre-commit install
+```
+
+Now, coding style checks will be enforced when you run `git commit`.
+
+[GDScript style guide]: https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html
+[godot-gdscript-toolkit]: https://github.com/Scony/godot-gdscript-toolkit
+[pre-commit]: https://pre-commit.com/
+
+## Git commit messages and pull request descriptions
```

Comment:
If you want to provide references to places that discuss this in more depth, you can consider:
https://blog.mocoso.co.uk/posts/talks/telling-stories-through-your-commits/
https://cbea.ms/git-commit/
https://groengaard.dev/blog/git-and-how-we-commit